### DISCUSSION TITLE
Per core governor setting (optional)

### DISCUSSION THREAD

**Discussion by Foosec** [AdnanHodzic/auto-cpufreq#547] [Ideas]
I wonder if a (optional) per core governor setting would be useful, this way one could for example set 2 cores to ondemand and 6 cores to powersave when on battery, perhaps prolonging battery life? I am unsure how the Linux scheduler would behave in this scenario, if it is aware and takes into account the core's governor when deciding where to put a task.

**Comment by shadeyg56** [comment]
It is (kind of) possible to set individual core governors, but the scheduler and cpu scaling run independently of each other i,e the scheduler has no idea what governor is being used. Linux already uses the `schedutil` governor by default, which adjusts the frequency according to the current demand for resources.
